### PR TITLE
[Estuary] Rating position fix + position adjustments for ListThumbInfoPanel & RightListPanel

### PR DESCRIPTION
**Pull request by jjd-uk**
## Description

Merged code in https://github.com/xbmc/xbmc/pull/17010 had wrong position for rating circle (star) for Music List and Now Playing views. This has been adjusted and took the opportunity to revisit rating star and plot positioning for TV Shows, again minor adjustments.

Additionally added rating circle for the current playing song in Now Playing.

**Before**
image

**After**
image

**Before**
https://i.imgur.com/JSpdtqQ.png
image

**After**
image

## Motivation and Context
Bug fix and some minor adjust to improve layout.

## How Has This Been Tested?
Tested on Windows x64 with variety of Music and TV Shows with and without User Ratings.

## Types of change

- [x ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [x ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x ] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [x] My change requires a change to the documentation, either Doxygen or wiki
- [x] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [x] All new and existing tests passed

### CODE REVIEW

File: addons/skin.estuary/xml/View_50_List.xml [hunk 1]
```diff
@@ -50,12 +50,12 @@
 						<height>850</height>
 						<bottom>124</bottom>
 						<fadetime>200</fadetime>
-						<aspectratio align="center" aligny="center">keep</aspectratio>
+						<aspectratio align="center" aligny="center">bottom</aspectratio>
```
Comment (ronie) [inline addons/skin.estuary/xml/View_50_List.xml]: i think you mean `aligny="bottom"` 
aspectratio should be `keep`

File: addons/skin.estuary/xml/View_50_List.xml [hunk 2]
```diff
@@ -65,18 +65,32 @@
 					<visible>Control.IsVisible(503) + Window.IsActive(musicplaylist)</visible>
 					<control type="image">
 						<left>58</left>
-						<top>120</top>
-						<width>540</width>
-						<height>470</height>
-						<aspectratio aligny="bottom">keep</aspectratio>
+					<top>110</top>
```
Comment (ronie) [inline addons/skin.estuary/xml/View_50_List.xml]: indenting

**Review by ronie — CHANGES_REQUESTED** [review]

**Comment by jjd-uk** [comment]
Updated to resolve comments.

**Comment by ronie** [comment]
thx!

**Pull request merged**